eng/Version.Details.xml

```diff
@@ -57,352 +57,316 @@
       <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>bc91b865ad4e746e7e3f281351e8d4a8d1dc80a0</Sha>
+      <Sha>8fb38ab52b8f873109b6e87b4f04226d6a892ee4</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging.TraceSource" Version="5.0.0-preview.2.20121.3" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
+    <Dependency Name="Microsoft.Extensions.ParameterDefaultValue.Sources" Version="5.0.0-preview.2.20126.1" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
       <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>bc91b865ad4e746e7e3f281351e8d4a8d1dc80a0</Sha>
+      <Sha>8fb38ab52b8f873109b6e87b4f04226d6a892ee4</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging.Testing" Version="5.0.0-preview.2.20121.3" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
+    <Dependency Name="Microsoft.Extensions.Primitives" Version="5.0.0-preview.2.20126.1" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
       <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>bc91b865ad4e746e7e3f281351e8d4a8d1dc80a0</Sha>
+      <Sha>8fb38ab52b8f873109b6e87b4f04226d6a892ee4</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging" Version="5.0.0-preview.2.20121.3" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
+    <Dependency Name="Microsoft.Extensions.TypeNameHelper.Sources" Version="5.0.0-preview.2.20126.1" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
       <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>bc91b865ad4e746e7e3f281351e8d4a8d1dc80a0</Sha>
+      <Sha>8fb38ab52b8f873109b6e87b4f04226d6a892ee4</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.ObjectPool" Version="5.0.0-preview.2.20121.3" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
+    <Dependency Name="Microsoft.Extensions.ValueStopwatch.Sources" Version="5.0.0-preview.2.20126.1" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
       <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>bc91b865ad4e746e7e3f281351e8d4a8d1dc80a0</Sha>
+      <Sha>8fb38ab52b8f873109b6e87b4f04226d6a892ee4</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Options.ConfigurationExtensions" Version="5.0.0-preview.2.20121.3" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
+    <Dependency Name="Microsoft.Internal.Extensions.Refs" Version="5.0.0-preview.2.20126.1" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
       <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>bc91b865ad4e746e7e3f281351e8d4a8d1dc80a0</Sha>
+      <Sha>8fb38ab52b8f873109b6e87b4f04226d6a892ee4</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Options.DataAnnotations" Version="5.0.0-preview.2.20121.3" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
```

Comment:
The diff is bad here because XML. The list of things I removed is based on the package list in 19033.

If there were still 'extra' references then the dependency update would fail, so I know I got them all.